src/pages/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Header from '../components/Header';
+import ListCountries from '../components/ListContry/ListCountries';
+
+const Home = () => (
+  <div className="max-width">
+    <Header name="Home" canBack={false} />
+    <ListCountries />
+  </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! The `Home` page should have a list of categories that could be filtered using given parameters, e.g. category name. Also keep [Behance layout design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) so add all `Highlight` icons and `header figure`(e.g Part of Europe on Behance design) provided picture below, please implement my suggestions 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/217588552-a280f004-663e-40a0-be70-e38df8b0b7f6.png)